content/en/containers/cluster_agent/setup.md

```diff
@@ -41,24 +41,52 @@ Then, upgrade your Datadog Helm chart.
 
 This automatically updates the necessary RBAC files for the Cluster Agent and Datadog Agent. Both Agents use the same API key.
 
-This also automatically generates a random token in a `Secret` shared by both the Cluster Agent and the Datadog Agent. You can manually set this by specifying a token in the `clusterAgent.token` configuration. You can also manually set this by specifying an existing `Secret` name containing a `token` value through the `clusterAgent.tokenExistingSecret` configuration. When set manually, this token must be 32 alphanumeric characters.
+This also automatically generates a random token in a `Secret` shared by both the Cluster Agent and the Datadog Agent to secure communication. You can manually specify this token used by setting the `clusterAgent.token` configuration. You can alternatively set this by referencing the name of an existing `Secret` containing a `token` value through the `clusterAgent.tokenExistingSecret` configuration.
```

Comment:
in `v2alpha1`:
- `clusterAgent.token` is `spec.global.clusterAgentToken `
- `clusterAgent.tokenExistingSecret` is `spec.global.clusterAgentTokenSecret`

Reply:
This section was actually for the Helm portion so keeping those configs as is. Had just updated this blurb to match the language of the Operator blurb, swapping out the config keys.